compliance/tests/struct_type_assertion/struct_type_assertion.gs.ts

```diff
@@ -7,14 +7,14 @@ export function main(): void {
 	let i: null | any = null
 	i = {Name: "Alice"}
 
-	let { value: s, ok: ok } = $.typeAssert<any/* unhandled type: *types.Struct */>(i, {kind: $.TypeKind.Struct, fields: new Set(['Name'])})
+	let { value: s, ok: ok } = $.typeAssert<any/* unhandled type: *types.Struct */>(i, {kind: $.TypeKind.Struct, fields: new Set(['Name']), methods: new Set()})
 	if (ok) {
 		console.log("Name:", s.Name)
 	} else {
 		console.log("Type assertion failed")
 	}
 
-	let { value: j, ok: ok2 } = $.typeAssert<any/* unhandled type: *types.Struct */>(i, {kind: $.TypeKind.Struct, fields: new Set(['Age'])})
+	let { value: j, ok: ok2 } = $.typeAssert<any/* unhandled type: *types.Struct */>(i, {kind: $.TypeKind.Struct, fields: new Set(['Age']), methods: new Set()})
```

Comment:
We should specify methods for struct as well as interfaces since struct can satisfy interfaces